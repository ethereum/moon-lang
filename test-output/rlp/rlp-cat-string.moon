either = zb2rhjDvcJ6d6xsRH5o6ZAsVmM7CGhRTZo2ADCQKtuXE4ijzy
encode = zb2rhhoXXkduCezoqZtAiNQ4Mggz1KDZBtyoagoEbRcjJEX5H
Right  = (either "Right")
dog    = (Right "cat")
(encode dog)

