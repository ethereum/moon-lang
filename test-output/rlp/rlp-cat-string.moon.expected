"0x83636174"
